### PR TITLE
Release notes for 1.15

### PR DESCRIPTION
**Pull request by maroshii**

### CODE REVIEW

**Review by codyjlandstrom — COMMENTED** [review]
Added some tweaks. Looks good!

File: src/content/self-hosted/install/releases.mdx [hunk 1]
```diff
@@ -7,14 +7,22 @@ id: releases
...
+22 November 2023
 
 This version is compatible with Kubernetes versions 1.25 to 1.27.
 
 ### Improvements
 
 - Upgrade ingress-nginx to 1.9.4 (Helm chart version 4.8.3)  <!-- 7386 -->
 - Enabled [chroot mode](https://kubernetes.io/blog/2022/04/28/ingress-nginx-1-2-0/) by default in both ingress-nginx controllers managed by Okteto. More info in [Upgrade your Okteto instance section](https://www.okteto.com/docs/self-hosted/install/upgrade/#upgrading-to-okteto-115x) <!-- 7425 -->
+- `okteto up` sessions are now garbage collected if no files have been synced for an hour   <!-- 7424 -->
+- Upgrade nginx to [1.8.4](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.9.4) (chart 4.8.3) <!-- 7386 -->
```
Comment (ifbyol) [inline src/content/self-hosted/install/releases.mdx:19]: This one is duplicated. It was one added by myself (line 16). Let's merge them to add the link to the release

File: src/content/self-hosted/install/releases.mdx [hunk 2]
```diff
@@ -7,14 +7,22 @@ id: releases
 
 ## 1.15.0
 
-TBD
+22 November 2023
 
 This version is compatible with Kubernetes versions 1.25 to 1.27.
```
Comment (ifbyol) [inline src/content/self-hosted/install/releases.mdx:12]: We need add a removal notice to indicate Kubernetes `1.24` is no longer supported. As we did [here](https://www.okteto.com/docs/self-hosted/install/releases/#removal-notice) for example

File: src/content/self-hosted/install/releases.mdx [hunk 3]
```diff
@@ -7,14 +7,22 @@ id: releases
...
 
 ### Improvements
```
Comment (ifbyol) [inline src/content/self-hosted/install/releases.mdx:14]: We have to include the version of the CLI included as part of the release. We usually include it as feature I think

File: src/content/self-hosted/install/releases.mdx [hunk 4]
```diff
@@ -7,14 +7,22 @@ id: releases
...
 
 - Upgrade ingress-nginx to 1.9.4 (Helm chart version 4.8.3)  <!-- 7386 -->
 - Enabled [chroot mode](https://kubernetes.io/blog/2022/04/28/ingress-nginx-1-2-0/) by default in both ingress-nginx controllers managed by Okteto. More info in [Upgrade your Okteto instance section](https://www.okteto.com/docs/self-hosted/install/upgrade/#upgrading-to-okteto-115x) <!-- 7425 -->
+- `okteto up` sessions are now garbage collected if no files have been synced for an hour   <!-- 7424 -->
+- Upgrade nginx to [1.8.4](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.9.4) (chart 4.8.3) <!-- 7386 -->
+
+### Bug Fixes
```
Comment (ifbyol) [inline src/content/self-hosted/install/releases.mdx:21]: I think there are 2 fixes missing:
* The one related to this commit `restore .gitmodules file after cloning`. I think this was generating some issue in the installer pods when there was submodules and images were being rebuilt always even if they were not needed cc @andreafalzetti could you provide more information?
* We have fix in the admin UI to show the proper configuration for the GC. cc @viddo Can provide more information

File: src/content/self-hosted/install/releases.mdx [hunk 5]
```diff
@@ -7,14 +7,22 @@ id: releases
...
+
+- Fix cli authentication when using openid "audience" or "resource" attributes  <!-- 7363 -->
```
Comment (ifbyol) [inline src/content/self-hosted/install/releases.mdx:23]: This one was already mentioned on `1.14.1` release notes. I think we don't usually include it in the next minor.

If so, we should also mention the fix for the support of registries with custom ports

File: src/content/self-hosted/install/releases.mdx [hunk 6]
```diff
@@ -7,14 +7,22 @@ id: releases
...
+- Fallback to default branch when destroying installer jobs  <!-- 7382 -->
+- Add scrollbar to space selector in the UI  <!-- 7355 -->
```
Comment (ifbyol) [inline src/content/self-hosted/install/releases.mdx:25]: Same about being already mentioned as part of `1.14.1` release

File: src/content/self-hosted/install/releases.mdx (hunk 4 above)
Comment (viddo) [reply src/content/self-hosted/install/releases.mdx:21]: I did include it in the original PR (7404), copy-paste that should have been included here:

> Fix wrong "Scale to Zero Period" value in the UI of Admin → Namespaces

Aside, figure you might want to look into why that wasn't extracted as expected?

Comment (andreafalzetti) [reply src/content/self-hosted/install/releases.mdx:21]: The fix I did on git submodules fixes the smart builds (or pull first) to work when git submodules is used and the `.gitmodules` is using `git`.

Related PR: https://github.com/okteto/app/pull/7406

For a release note I would write:

```
- Bug fix: smart builds work on repositories using `.gitmodules` containing submodules cloned via ssh key
```

**Comment by maroshii** [comment]
All feedback has been addressed. Merging as-is. If there's anything else, I'll included as a follow up in the versioned docs PR and oficial release tomorrow

File: src/content/self-hosted/install/releases.mdx [hunk 7]
```diff
@@ -26,6 +37,11 @@ This version is compatible with Kubernetes versions 1.24 to 1.27.
 
 - Fix private registry credentials integration for registries with custom ports (other that the default 443)
 
+### Features
+
+- The Okteto CLI used for Pipeline installation has been upgraded to version [2.22.3](https://github.com/okteto/okteto/releases/tag/2.22.3)
```
Comment (ifbyol) [inline src/content/self-hosted/install/releases.mdx:42]: @maroshii This was added in the wrong version, right? CLI `2.22.x` is included as part of `1.15.0` and not as part of `1.14.2`, right?

**Review by ifbyol — COMMENTED** [review]
Post-merge review

**Pull request merged**